### PR TITLE
docs - add pxf json profile info

### PR DESCRIPTION
**Pull request by lisakowen**
pxf now supports accessing JSON format data stored in HDFS:
- include JSON info on HDFS page (rather than document as a separate connector)
- examples for json records that span single and multiple lines

doc review site links:
- http://docs-gpdb-review-staging.cfapps.io/510/pxf/hdfs_read_pxf.html#profile_hdfsjson
- a few minor changes in other files

### CODE REVIEW

File: gpdb-doc/markdown/pxf/hdfs_read_pxf.html.md.erb [hunk 1]
```diff
@@ -479,5 +482,211 @@ Perform the following steps to create and query an external table referencing th
...
+## <a id="profile_hdfsjson"></a>Reading JSON Data
+
+Use the `Json` profile when you want to read native JSON format data from HDFS.
+
+
+### <a id="hdfsjson_work" class="no-quick-link"></a>Working with JSON Data
+
+JSON is a text-based data-interchange format. JSON data is typically stored in a file with a `.json` suffix. 
+
+A `.json` file will contain a collection of objects. A JSON object is a collection of unordered name/value pairs. A value can be a string, a number, true, false, null, or an object or array. You can define nested JSON objects and arrays.
```
Comment (denalex) [inline gpdb-doc/markdown/pxf/hdfs_read_pxf.html.md.erb]: English is my second language and article use is the weakest spot :-) I wonder why do we not put "an" before "array" word in the fragment "..., or an object or array" above ?

Comment (lisakowen) [reply gpdb-doc/markdown/pxf/hdfs_read_pxf.html.md.erb]: english is my first language and as you can see, i don't catch everything :^).  i will add the missing "an".

File: gpdb-doc/markdown/pxf/hdfs_read_pxf.html.md.erb [hunk 2]
```diff
@@ -479,5 +482,211 @@ Perform the following steps to create and query an external table referencing th
...
+$ hdfs dfs -mkdir /data/pxf_examples
+$ hdfs dfs -put singleline.json /data/pxf_examples
+$ hdfs dfs -put multiline.json /data/pxf_examples
+```
+
+Once the data is loaded to HDFS, you can use Greenplum Database and PXF to query and analyze the JSON data.
+
+### <a id="json_customopts" class="no-quick-link"></a>Custom Options
+
+PXF supports single- and multi- line JSON records. When you want to read multi-line JSON records, you must provide an `IDENTIFIER` \<custom-option\> and value. Use this \<custom-option\> to identify to PXF the member name of the first field in the JSON record object:
```
Comment (dyozie) [inline gpdb-doc/markdown/pxf/hdfs_read_pxf.html.md.erb]: remove "to PXF"

File: gpdb-doc/markdown/pxf/hdfs_read_pxf.html.md.erb [hunk 3]
```diff
@@ -479,5 +482,211 @@ Perform the following steps to create and query an external table referencing th
...
+
+To query the JSON data in the external table:
+
+``` sql
+SELECT * FROM singleline_json_tbl;
+```
+
+### <a id="jsonexample2" class="no-quick-link"></a>Example: Multi-Line Records
+
+The `CREATE EXTERNAL TABLE` SQL command to create a readable external table referencing the multi-line-per-record JSON data set is very similar to that of the single line data set above. You must additionally specify the `LOCATION` clause `IDENTIFIER` keyword and an associated value when you want to read multi-line JSON records. For example:
```
Comment (dyozie) [inline gpdb-doc/markdown/pxf/hdfs_read_pxf.html.md.erb]: This sentence is a bit awkward.  Would rewrite it to "Teh SQL command to create a readable external table from multi-line-per-record JSON file is very similar ..."

**Comment by lisakowen** [comment]
thanks for reviewing @shivzone @denalex @lavjain @dyozie!

**Pull request merged**